tools/lib/tests/test_caching.py

```diff
@@ -1,18 +1,24 @@
 #!/usr/bin/env python3
 import os
-import shutil
 import unittest
+import tempfile
+from unittest import mock
 
-os.environ["COMMA_CACHE"] = "/tmp/__test_cache__"
-from openpilot.tools.lib.url_file import URLFile, CACHE_DIR
+from openpilot.tools.lib.url_file import URLFile
 
 
 class TestFileDownload(unittest.TestCase):
+  def setUp(self):
+    self.temp_dir = tempfile.TemporaryDirectory()
+    self.cache_dir_patch = mock.patch("openpilot.tools.lib.url_file.CACHE_DIR", self.temp_dir.name)
+    self.cache_dir_patch.start()
```

Comment:
seems like something that we'll want to use in a lot of places. I'd setup a decorator to do this

Reply:
done